src/SonarQube.Client/Services/SonarQubeService.cs

```diff
@@ -66,37 +67,37 @@ public bool HasOrganizationsFeature
                 throw new InvalidOperationException("This operation expects the service not to be connected.");
             }
 
-            var connectionDto = new ConnectionDTO
+            var ConnectionRequest = new ConnectionRequest
             {
                 Authentication = connection.Authentication,
                 ServerUri = connection.ServerUri,
                 Login = connection.UserName,
                 Password = connection.Password
             };
 
-            var credentialsValidationResult = await this.sonarqubeClient.ValidateCredentialsAsync(connectionDto, token);
+            var credentialsValidationResult = await this.sonarqubeClient.ValidateCredentialsAsync(ConnectionRequest, token);
             credentialsValidationResult.EnsureSuccess();
 
-            if (!credentialsValidationResult.Value.AreValid)
+            if (!credentialsValidationResult.Value.IsValid)
             {
                 throw new Exception("Invalid credentials."); // TODO: Provide better exception
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 'System.Exception' should not be thrown by user code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS112)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)